index.html

```diff
@@ -3186,12 +3186,20 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-The value of the <code>termsOfUse</code> <a>property</a> tells the
-<a>verifier</a> what actions it is required to perform (an <em>obligation</em>),
+Examples for the value of the <code>termsOfUse</code> <a>property</a> may be
+to tells the <a>verifier</a> 
+<li>
+- what actions it is required to perform (an <em>obligation</em>),
```

Comment:
```suggestion
what actions it is required to perform (an <em>obligation</em>),
```

Reply:
@TallTed I have made the editorial changes you have requested, and the revised example has been taken from the EBSI documentation and has a real URL that can be resolved.